powerquery-docs/connectors/cloudbluepsa.md

```diff
@@ -0,0 +1,83 @@
+title: Power Query CloudBlue PSA connector
+description: Provides basic information and prerequisites for the operation of connector in conjuction with the API.
+author: bezhan-msft
+
+
+ms.topic: conceptual
+ms.date: 07/03/2022
+ms.author: bezhan 
+LocalizationGroup: reference
+---
+
+# CloudBluePSA
+
+>[!Note]
+The following connector article is provided by CloudBluePSA, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the CloudBluePSA website and use the support channels there.
+
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
+| Products | Power BI (datasets)
+| Authentication Types Supported | API Key |
+| Function Reference Documentation | [CloudBluePSA Connector.Help](https://help.harmonypsa.com/articles/#!cloudblue-psa-4-28-publication/using-the-power-bi-connector)
+| | |
+## Prerequisites
+Before you can use the CloudBluePSA connector, you must have an CloudBlue PSA instance (username/password) and an API key.
+
+
+## Capabilities
+* Import
+
+## Connect to CloudBlue PSA
+
+To connect to CloudBlue PSA data:
+
+1. Select **Get Data** from the **Home** ribbon in Power BI Desktop. 
+
+2. Click in the **Search** box and start typing **CloudBluePSA**, then select **CloudBluePSA** from the list on the right, and select **Connect**.
+
+    ![Get Data from CloudBlue PSA.](./media/cloudbluepsa/getdata.png)
+
+
+3. In the **Retrieve all pages of data** window that appears, copy and paste the **URL** generated on the **GET** side of the API endpoint of your choice, and below on the **Filter** dialog box, copy and paste the constructed filter on the same API endpoint. For example:
+
+    ### URL: ### 
+    https://INSTANCE_URL/webapi/v1.3/tickets/getticketsreport
+    
+    ### Filter ### 
+    {  "pageno": 1,  "pagesize": 10,  "filters": [{"field": "Type", "op": "cn", "data": "n"}], "filterop": "AND",  "sortcolumn": "Name",  "sortorder": "Asc",  "resulttype": "Json",  "includeccysymbols": true,  "gridcolumns": "RefNumber,Type,Name"}
+
+    ![Retrieve all pages of data from CloudBlue PSA.](./media/cloudbluepsa/retrieveallpagesofdata.png)
+
+3. Click **OK**.
+
+4. Provide your **API key** generated in CloudBluePSA and select **Connect**.
+
+5. Select **Transform Data**.
+
+6. Expand the **Data** column, then selecting **OK**.
+
+    ![Expand Data Column.](./media/cloudbluepsa/expand1.png)
+
+
+7. Expand the **Column1.1** column. This time, on the list of columns all the grid columns included in the filter are displayed. Select as many columns as required then select **OK**. 
+All the selected data is now displayed, and can be reshaped and used to create reports as required.
+
+    ![Expand Data Column1.1.](./media/cloudbluepsa/expand1.1.png)
+
+
+8. Select **Close & Apply**. You can now start using your data.
+
+ 
```

Comment:
This needs a Connect to CloudBlue PSA in Power Query Online section.

Reply:
Hi Bob, I am not sure what to add here. Could you please provide an example? Thank you

Reply:
You can use the [Azure Blob Storage](https://learn.microsoft.com/power-query/connectors/azure-blob-storage) "Connect to Azure Blob Storage from Power Query Online" section as an example.

Reply:
Did you follow the instructions in the Power Query documentation article called [Where to get data](https://learn.microsoft.com/power-query/where-to-get-data), specifically the section titled [Get data in Power BI service](https://learn.microsoft.com/power-query/where-to-get-data#get-data-in-power-bi-service)? If not, start there, and you should be able to run Power Query Online.

If you can't access the Power BI service, just use the [free trial version](https://powerbi.microsoft.com/getting-started-with-power-bi/). Go to that link, then select "Free trial" in the upper right hand corner.

Reply:
The changes for this are now pushed.